tests/integration/thrift/test_tornado_client.py

```diff
@@ -82,7 +83,7 @@ def test_thrift_exception(mock_server, thrift_service):
         'thrift',
         method='getItem',
     ).and_raise(thrift_service.ItemDoesNotExist("stahp"))
-
+    mock_server.tchannel.hooks.register(ZipkinTraceHook(tchannel=mock_server))
```

Comment:
The existing test has this mock_trac_record.called assertion. I guess it is testing if trace will be triggered when exception happens in thrift. This is your test. If it doesn't mean that. I will delete it.